vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/check_service_retired.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Description: This method checks to see that all of the service resources are retired before retiring the service.
+#
+
+$evm.log("info", "Listing Root Object Attributes:")
+$evm.root.attributes.sort.each { |k, v| $evm.log("info", "\t#{k}: #{v}")  }
+$evm.log("info", "===========================================")
+
+service = $evm.root['service']
+if service.nil?
+  $evm.log('info', "Service Object not found")
+  exit MIQ_ABORT
```

Comment:
Use a `fatal` logging message here if we are exiting the state machine with `MIQ_ABORT`.